src/ports/postgres/modules/utilities/test/pivot.sql_in

```diff
@@ -149,7 +149,7 @@ SELECT pivot('pivset_ext', 'pivout', 'id, id2', 'piv, piv2', 'val, val2',
 SELECT * FROM pivout;
 
 SELECT assert(__p_7__ = 1.5,
```

Comment:
Install-check fails at this point. Since the new dictionary starts from 1 (not 0) the __p_8__ column has this value.